src/map/clif.c

```diff
@@ -9338,7 +9341,7 @@ void clif_parse_LoadEndAck(int fd, struct map_session_data *sd) {
 	sd->state.warping = 0;
 	sd->state.dialog = 0;/* reset when warping, client dialog will go missing */
 
-	// look
+	//< Character Looks
```

Comment:
There's an extra `<` sign here. Please note the syntax recognized by doxygen:

```
int foo; ///< Description of foo (three slashes and a left arrow to point to the variable declaration to the left)
```

or

```
/// Description of foo (three slashes and no arrow to refer to the following declaration)
int foo;
```

or

```
/**
 * Description of foo
 * (c-style block comment starting with slash star star, also refers to the following declaration)
 */
int foo;
```